specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ContainerApps.json

```diff
@@ -756,6 +766,41 @@
         }
       }
     },
+    "IpSecurityRestrictionRule": {
+      "description": "Rules to restrict incoming IP address.",
+      "type": "object",
+      "required": [
+        "name",
+        "ipAddress",
+        "action"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name for the IP restriction rule.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Describe the IP restriction rules that are being sent to the container-app. This is an optional field.",
+          "type": "string"
+        },
+        "ipAddress": {
```

Comment:
>ipAddress [](http://example.com/codeflow?start=9&length=9)

Can we change the name to ipAddressRange? I know it is not consistent with appService but that is more correct name (appservice just could not change the name after API was released). If this is a too big change then keeping as it is ok.

Reply:
renamed